### DISCUSSION TITLE
export ambf model to unity

### DISCUSSION THREAD

**Discussion by Debora-b** [WPI-AIM/ambf#171] [Q&A]
hi! 
there is a way to transfer the scene captured by the camera of the endoscope into a unity project?
I'm using two different computer and  since i want a dinamic communication i'm using Rosbridge, in this way i'm able to connect the two computer and share msgs between Unity(windows) and ROS,  now i need to visualize that scene on unity, is it possible? how could i do this?
thank you in advanced

**Comment by adnanmunawar** [comment]
Thanks for creating the issue. I have not used Unity myself so at this time I do not know how to make this specific scenario work. However, I shall ask around my colleagues who have experience with Unity and ROS and will update on this thread.
